src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <section class="row">
+      <div class="row__item">
+        <div class="card" data-qa="card">
+          <a href="#" class="card__link">
+            <img
+              class="card__image" src="images/imac.jpeg"
+              alt="Моноблок Apple A1419 iMac 27"
+              width="160px" height="134px"
+            >
+          </a>
+          <h3 class="card__title">
+            Моноблок APPLE A1419 iMac 27&#34; Retina 5K (MNED2UA/A)
+          </h3>
+          <p class="card__code">Код товара: 195434</p>
+          <div class="card__stars">
+            <div class="stars stars--4">
```

Comment:
```suggestion
          <div class="card__stars stars stars--4">
```
можно испбавться от лишнего div